src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-cancel-dialog.component.html

```diff
@@ -13,22 +13,19 @@
 <!--    </ng-template>-->
     <div class="modal-header">
         <h5 class="modal-title mb-1" data-cy="therapeuticRegimeCancelDialogHeading" jhiTranslate="supportivecareApp.therapeuticRegime.cancel.title">Deseja sair da criação de um regime terapêutico?</h5>
-
-        <button type="button" class="close" data-dismiss="modal" aria-hidden="true"
-                (click)="cancel()">&times;</button>
     </div>
 
     <div class="modal-body">
         <p id="custom-cancel-therapeuticRegime-heading" jhiTranslate="supportivecareApp.therapeuticRegime.cancel.question">Are you sure you want to cancel this Therapeutic Regime?</p>
     </div>
 
     <div class="modal-footer">
-        <button type="button" class="btn btn-secondary" data-dismiss="modal" (click)="cancel()">
-            <fa-icon icon="ban"></fa-icon>&nbsp;<span jhiTranslate="entity.action.cancel">Cancel</span>
+        <button id="custom-confirm-cancel-therapeuticRegime" data-cy="entityConfirmCancelButton" [routerLink]="['/therapeutic-regime']" (click)="cancel()" type="button" class="btn btn-danger">
```

Comment:
Would be better to call a specific function om the component, because this modal could be used in other pages...